src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -65,6 +58,45 @@ class IOUAmountPage extends React.Component {
         }
     }
 
+    /**
+     * Update amount with number or Backspace pressed.
+     * Validate new amount with decimal number regex up to 6 digits and 2 decimal digit to enable Next button
+     *
+     * @param {String} key
+     * @param {String} event
+     */
+    onlyAllowNumericValues(key, event) {
+        // Prevent reusing synthetic event object, so we can call preventDefault() later.
+        if (event) {
+            event.persist();
+        }
+
+        // Backspace button is pressed
+        if (key === '<' || key === 'Backspace') {
```

Comment:
What is the `<` key?

Reply:
In BigNumberPad component that we used in mobile, we have `<` for backspace button.